src/OrleansSQLUtils/Storage/DbStoredQueries.cs

```diff
@@ -192,14 +192,18 @@ internal static ReminderEntry GetReminderEntry(IDataRecord record)
             /// </summary>
             private static string TryGetSiloName(IDataRecord record)
             {
-                for (var i = 0; i < record.FieldCount; ++i)
```

Comment:
@xclayl A tangential that might make sense (but can also be refactored in another PR if it makes sense)
> Another option is to return the columns with the upper-lower case originally expected. But just out of curiosity, why does this column work differently than the others? ie "TryGet..."

```
/// <summary>
        /// Returns a value if it is not <see cref="System.DBNull"/>, <em>default(TValue)</em> otherwise.
        /// </summary>
        /// <typeparam name="TValue">The type of the value to request.</typeparam>
        /// <param name="record">The record from which to retrieve the value.</param>
        /// <param name="fieldName">The name of the field to retrieve.</param>
        /// <param name="default">The default value if value in position is <see cref="System.DBNull"/>.</param>
        /// <returns>Either the given value or the default for the requested type.</returns>
        /// <exception cref="IndexOutOfRangeException"/>
        /// <remarks>This function throws if the given <see paramref="fieldName"/> does not exist.</remarks>
        public static TValue GetValueOrDefault<TValue>(this IDataRecord record, string fieldName, TValue @default = default(TValue))
        {
            var ordinal = record.GetOrdinal(fieldName);
            return record.IsDBNull(ordinal) ? @default : (TValue)record.GetValue(ordinal);
}
```
https://github.com/dotnet/orleans/blob/master/src/OrleansSQLUtils/Storage/DbExtensions.cs#L117